packages/forms/src/UIForm/utils/widgets.js

```diff
@@ -56,7 +56,7 @@ const widgets = {
 	code_text: CodeTextMode,
 	comparator_text: TextModeComparator,
 	datalist_text: DatalistTextMode,
-	file_text: () => null,
+	file_text: FileTextMode,
```

Comment:
Not a breaking change imho, you simply show what was previously hidden

Reply:
Thanks buddy. I'm happy to not call it a BC if that's the consensus. I suppose I look at this kind of thing as if it's a new feature which forces an unrequested change in behaviour then it is breaking to the experience, even if it doesn't cause errors.

Reply:
Actually, imagine CSS changes, they could be UI/UX huge changes but the apps will continue to work without any developer manipulations 🤷

Reply:
Agree, which is why I consider CSS changes to be breaking too! :grin:

I suppose it's just the definition of 'breaking' as 'causes errors' or 'impacts the experience'. I suppose my view is a result of my background :shrug: